### ISSUE TITLE
Add plotting for PPO run

### ISSUE THREAD

**Issue by vlad17** [mwhittaker/deeprl_project#16]
Deliverable: plot.py similar to what we had in the assignments that takes a log directory generated by PPO and spits out a learning rate curve.

[plot.py.zip](https://github.com/mwhittaker/deeprl_project/files/1418673/plot.py.zip)

**Comment by SamvitJ** [comment]
Deliverable: short learning curve for video PPO, trained on only a couple of training steps

**Comment by vlad17** [comment]
Blocked on #30 (to see if CPU is faster)
